gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -257,15 +270,34 @@ public GrpcCallContext withTimeout(@Nullable Duration timeout) {
 
   @Nullable
   @Override
-  public Duration getTimeout() {
+  @ObsoleteApi("Use getTimeoutDuration() instead")
+  public org.threeten.bp.Duration getTimeout() {
+    return toThreetenDuration(getTimeoutDuration());
+  }
+
+  @Nullable
+  @Override
+  public java.time.Duration getTimeoutDuration() {
     return timeout;
   }
 
+  /**
+   * Backport of {@link #withStreamWaitTimeoutDuration(java.time.Duration)} using {@link
+   * org.threeten.bp.Duration}
+   */
+  @Override
```

Comment:
same comment as above - add this to the Javadoc as well

Reply:
Done